### ISSUE TITLE
[Reproduce] Cannot Reproduce the results.

### ISSUE THREAD

**Issue by wjun0830** [Verg-Avesta/CounTR#1]
Hello. Thank you for the great work.

However, under the given instructions with the pretrained weight, I could not reproduce the reported result on FSC-147.

Actually, due to the error regarding 'torch._six container_abcs', I could not run the evaluation code with torch==1.10.
(Same error occurs even with 3090Ti server)

Then, when I tried to lower the torch version to 1.8 which is stable, the performance on the test set becomes  "MAE : 14.98, MSE : 106.51".

Is there any way to fix this issue and reproduce the reported results?
Or, if may, can you provide the docker image?

**Comment by Verg-Avesta** [comment]
Sorry to make you confused. These problems are caused by my negligence.

About your first problem of pytorch 1.10, we used [`timm==0.3.2`](https://github.com/rwightman/pytorch-image-models) in our job, for which a [fix](https://github.com/rwightman/pytorch-image-models/issues/420#issuecomment-776459842) is needed to work with PyTorch 1.8.1+. I have added this in the readme.md file. The change of the environments' version usually cause the result to change about 0.3 in MAE.

About your second problem of the low performance, I apologize for uploading a wrong test file used for zero-shot counting without test-time augmentation. I have also uploaded the right test file for few-shot counting with test-time augmentation, and you can check whether the results can be reproduced with it. 

If you can reproduce the corresponding results, you can close this issue. If there are still other problems, you can continue to contact me.

Best wishes.
